src/Gui/DlgSettingsNavigation.ui

```diff
@@ -512,7 +576,7 @@ The value is the diameter of the sphere to fit on the screen.</string>
       <item row="6" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
-         <string> Zoom step</string>
+         <string>Zoom step</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
```

Comment:
Something is not quite right with the .ui file as it triggers this warning:
```
AutoUic: FreeCAD/src/Gui/DlgSettingsNavigation.ui: Warning: The name 'horizontalSpacer' (QSpacerItem) is already in use, defaulting to 'horizontalSpacer1'.
/home/werner/Projects/FreeCAD/src/Gui/DlgSettingsNavigation.ui: Warning: The name 'label' (QLabel) is already in use, defaulting to 'label1'.
/home/werner/Projects/FreeCAD/src/Gui/DlgSettingsNavigation.ui: Warning: The name 'horizontalLayout' (QHBoxLayout) is already in use, defaulting to 'horizontalLayout1'.
```

Reply:
I had to rebuild to see this message. Probably why I have not noticed it before.